### PR TITLE
Update 2019-04-16-containers-with-docker.md

### PR DESCRIPTION
**Pull request by ghost**
Small fix on two code snippets.

### CODE REVIEW

File: src/_posts/2019-04-16-containers-with-docker.md [hunk 1]
```diff
@@ -252,7 +252,8 @@ Finally, we can run the application by passing the `CMD` to start
 
 ```Dockerfile
 FROM adoptopenjdk/openjdk11-openj9
-WORKDIR /realworldCOPY . /realworld
+WORKDIR /realworld
+COPY . /realworld
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-04-16-containers-with-docker.md:256]: I would do `COPY . .` so that you don't have to define the working directory in two places.

Comment (ghost) [reply src/_posts/2019-04-16-containers-with-docker.md:256]: I'm just fixing the typo on the original code snippet, maybe Andre could intervene on that?

Comment (ghost) [reply src/_posts/2019-04-16-containers-with-docker.md:256]: `COPY . .` will also work in this case. I don't mind having that changed.

Comment (ghost) [reply src/_posts/2019-04-16-containers-with-docker.md:256]: IMHO, I think that keeping it the way you initially wrote it documents your progress as well.

**Pull request merged**